src/App.tsx

```diff
@@ -15,36 +16,110 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum ESortType {
+  ALPHABETICALLY = 'alphabetically',
+  LENGTH = 'by length',
```

Comment:
usually we use snake case. But I'd make it simpler and just keep `alphabet/length` in both enum keys and values

```suggestion
  LENGTH = 'by_length',
```